hknweb/templates/base.html

```diff
@@ -101,7 +101,7 @@
       <span class="slash">/</span>
       <a href="">Course Guide</a>
       <span class="slash">/</span>
-      <a href="">Department Tours</a>
+      <a href="/tours">Department Tours</a>
```

Comment:
You already have a name for these, so doing something like `{% url 'tours' 'index' %}` I believe should work here and make this more dynamic